UI/src/view/linear_asset/laneModellingForm.js

```diff
@@ -554,7 +554,7 @@
       return $('<div class="wrapper read-only">' +
           '   <div class="form form-horizontal form-dark asset-factory">' +
           '     <div class="form-group">' +
-          '       <p class="form-control-static asset-log-info">Lis&auml;tty j&auml;rjestelm&auml;&auml;n: ' + self.informationLog(info.createdDate, info.createdBy)+ '</p>' +
+          '       <p class="form-control-static asset-log-info">Lisätty järjestelmään: ' + self.informationLog(info.createdDate, info.createdBy)+ '</p>' +
```

Comment:
This will print the "Lisätty järjestelmään" correctly? Without any strange characters ?
If you look into dynamicAssetForm.js line 1192, is there this name encoded and make sense to be encoded since this "Nordic"/Special characters aren't standard so we need to encode it for html can parse it correctly.

Reply:
That's true and I feared that as well when i changed it but works ok, it prints correctly.

Reply:
I think yes.
We already have cases where we use normally the sentences with tremas without any problems I don't understand why it's used with encoding at the first time...

Like for example:
```Javascript
- var addLeftLane = $('<li>').append($('<button class="btn btn-secondary">Lisää kaista vasemmalle puolelle</button>').click(function() {

- '    <option class="empty" disabled selected>Lisää voimassaoloaika</option>' +

Reply:
The enconding is needed or a nice thing to have because in same way you need to have the code repository in encoding windows-1552 to support this tremas.
If you don't encond this characters for HTML you can have issues to show this characters  to some users.
If you want a detail explanation, where one example:
https://text-symbols.com/html/encode/